shipstation_integration/public/js/tag_list.js

```diff
@@ -0,0 +1,12 @@
+frappe.listview_settings["Tag"] = {
+    refresh: function (listview) {
+        listview.page.add_inner_button("Sync Tags from ShipStation", function () {
+            frappe.call({
+                method: 'shipstation_integration.tags.list_tags',
+                callback: function (r) {
+                    listview.refresh();
```

Comment:
This doesn't trigger a recursion loop, right? Should the function trigger move from `refresh` to `onload` instead since the button doesn't have any conditions for being loaded up?